workflows/prognostic_c48_run/runtime/steppers/reservoir.py

```diff
@@ -1,14 +1,29 @@
 import cftime
 import dataclasses
-from datetime import timedelta
+import logging
 import pandas as pd
-from typing import Optional, MutableMapping, Hashable, Mapping, cast
 import xarray as xr
+from datetime import timedelta
+from typing import Optional, MutableMapping, Hashable, Mapping, cast, Sequence, Dict
 
 import fv3fit
-from fv3fit._shared import put_dir
 from fv3fit._shared.halos import append_halos_using_mpi
 from fv3fit.reservoir.adapters import ReservoirDatasetAdapter
+from runtime.names import SST, MASK
+from .prescriber import sst_update_from_reference
+
+
+logger = logging.getLogger(__name__)
+
+
+RESERVOIR_SST = "sst"
+RESERVOIR_NAME_TO_STATE_NAME = {RESERVOIR_SST: SST}
```

Comment:
The renaming adapter doesn't quite work since predictor doesn't include the extra methods that are required for reservoir handling that exist on our adapter.  Rather than write an equivalent and increase the layers of abstraction from our original model, I opted to incorporate a more flexible renaming (like the adapter) but just applied in our steppers since that's the only point where we currently will need renaming for fv3 interaction. Hopefully this will cover our use cases